src/preset_cli/api/clients/superset.py

```diff
@@ -394,7 +394,7 @@ def get_resource(self, resource_name: str, resource_id: int) -> Any:
         response = self.session.get(url)
         validate_response(response)
 
-        resource = response.json()
+        resource = response.json()["result"]
```

Comment:
Are we okay with throwing a KeyError if result isn't available?

Reply:
Yeah, if the key is not there it's because the Superset API has completely changed, so it's OK. Also, today we'd also raise `KeyError`, only later in the code.